docs/UserGuide.md

```diff
@@ -296,19 +296,29 @@ Refer to [properties of contact](#properties-of-contact) on the accepted formats
 
 ###  Update contact's photo: `updatephoto`
 
-Updates the photo of an existing contact.
+If you are afraid that you may forget who the saved contact is in CampusConnect, our application enables you to update the photo of your contact so that you can visually remember and recognize them!
 
-Format: `updatephoto INDEX path/NEW_PHOTO_PATH`
+This feature involves the command: `updatephoto`, which updates the photo of an existing contact.
 
-Examples:
-* `updatephoto 1 path/C:/photos/new_johndoe.jpg`
-  * Updates the photo for the 1st person with a new image from the specified path.
-* `updatephoto 2 path/D:/images/new_betsycrowe.png`
-  * Updates the photo for the 2nd person with a new image from the specified path.
+Format: `updatephoto PERSON_INDEX [path/NEW_PHOTO_PATH]`
+
+<box type="info" icon=":fa-solid-magnifying-glass:">
+
+Below are some examples on how to use the command:
+* `updatephoto 1 path/C:/photos/new_johndoe.jpg`: Updates the photo to the contact at index 1 with a new image from the specified path.
+* `updatephoto 2 path/D:/images/new_betsycrowe.png`: Updates the photo to the contact at index 2 with a new image from the specified path.
+
+</box>  
 
 <box type="warning">
 
-You are required to input the absolute path to the photo for this command. If you use an invalid path, a default photo will be chosen for your specified contact.
+* You should provide a positive integer that is smaller than or equal to the number of contacts currently displayed in CampusConnect for `PERSON_INDEX`. 
+* You can get the path to the photo by using the following steps: 
+  * Right click on the photo that you want to use.
+  * Choose the option: Copy as path (on Windows) or Copy "name_of_file" as Pathname (on MacOS).
+  * After pasting the copied path in CampusConnect, remember to remove the double quotation marks!
+
+  ![photo_path](images/photopath.png)
```

Comment:
Should this be under the warning section? Perhaps you can shift it to after the line `format` to introduce to users how to get `NEW_PHOTO_PATH`